legend-engine-testable/src/main/java/org/finos/legend/engine/testable/assertion/TestAssertionHelper.java

```diff
@@ -29,7 +30,7 @@
 {
     protected static AssertionStatus compareAssertionJSON(TestAssertion parentAssertion, String _expected, String _actual) throws IOException
     {
-        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectMapper objectMapper = new ObjectMapper().enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
```

Comment:
Should this be added to PureProtocolObjectMapperFactory and we use objectMapper from that factory? What do you think?

Reply:
Don't think that's required in JSON comparison